mqtt-cs/src/main/java/org/apache/rocketmq/mqtt/cs/session/match/MatchAction.java

```diff
@@ -120,9 +120,7 @@ public void addSubscription(Session session, Set<Subscription> subscriptions) {
             }
 
             synchronized (topicCache) {
-                if (!topicCache.containsKey(topicFilter)) {
```

Comment:
This place is for subscribe request, that will not be frequent, can I keep it? @pingww